dev/request-runner.js

```diff
@@ -0,0 +1,10 @@
+import { API } from './mocks.js'
+import { writeFile, readFile } from 'fs/promises'
+
+const args = JSON.parse(process.argv[2])
+const db = `/tmp/${args.hash}.kv.json`
+NAN.entries = await readFile(db, 'utf8').then(JSON.parse, () => ({}))
```

Comment:
use the hash as db name to avoid conflict in database.
We must reload the database on every request, we will see if performances became an issue at some point